data/GRND/data.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Token Name",
+  "symbol": "SYMBOL",
+  "decimals": 18,
+  "description": "The token of the UNDRGRND Art Community",
+  "website": "https://undrgrnd.io",
+  "twitter": "@undrgrnd_nft",
+  "tokens": {
+    "ethereum": {
+      "address": "0x9816082D089faD585B0735dcaf15147bBB9A4C4d"
+    },
+    "optimism": {
+      "address": "0x337919Bc5737D046Dd4b559D2663aF7B0948EfE6"
+    },
+    "polygon": {
```

Comment:
can you remove this polygon entry? Polygon is not apart of the superchain

Please run the validation script locally and ensure it's passing so that this feedback loop is a but faster

Reply:
Polygon removed. We should be good to go!

Reply:
I'm not sure where the validation script is - not to waste too much of your time, but if there's a link to it I am happy to run it. I'm working on doing this for $PAGE as well, and using the Base bot for the first time, so just learning this process.

Reply:
nw! this is good signal to get our readme updated. Noticed the section about validation doesn't specify the validation command. I'll get that updated

Reply:
Few more things to fix
```
2024-05-01T03:08:59.5147639Z error: GRND on chain ethereum token 0x9816082D089faD585B0735dcaf15147bBB9A4C4d has incorrect symbol
2024-05-01T03:08:59.5149563Z error: GRND on chain ethereum token 0x9816082D089faD585B0735dcaf15147bBB9A4C4d has incorrect name. Got GRND
2024-05-01T03:08:59.5151770Z error: GRND on chain optimism token 0x337919Bc5737D046Dd4b559D2663aF7B0948EfE6 has incorrect symbol
2024-05-01T03:08:59.5153278Z error: GRND on chain optimism token 0x337919Bc5737D046Dd4b559D2663aF7B0948EfE6 has incorrect name. Got GRND
2024-05-01T03:08:59.5154525Z error: GRND on chain base token 0x7E00780C65D0291E5210fC208ABce0C5A966d538 has incorrect symbol
2024-05-01T03:08:59.5155646Z error: GRND on chain base token 0x7E00780C65D0291E5210fC208ABce0C5A966d538 has incorrect name. Got GRND
```

The data fields need to match what's onchain, with overrides set if that's not possible

Reply:
Got it. I had left in the templated Name and Symbol fields at the top, as a result of my effort to locate the formatting error. Setting those to Name and Symbol respectively did not work well with the token named GRND with a symbol of GRND. My bad! We should be good now though.